kinder/doc/kind-kinder.md

```diff
@@ -56,7 +56,6 @@ _Actions on a running cluster:_
         - shortcut for testing different kubeadm join discovery mechanics
     - `kubeadm init` can be executed as a unique workflow or using phases
     - `kubeadm join` can be executed as a unique workflow or using phases
-    - the init action installs Calico as a CNI plugin instead of kindnet
```

Comment:
Let's keep a note we are using kindnet, which is acceptable given that kubeadm should be CNI agnostic and validation of CNI providers is out of scope of our current E2E testing